### PR TITLE
EMP-517 missing partner fields

### PR DESCRIPTION
**Pull request by solomonmanik**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/LASB-XXX)

Describe what you did and why.

## Checklist

Before you ask people to review this PR:

- [ ] Tests should be passing: `./gradlew test`
- [ ] Github should not be reporting conflicts; you should have recently run `git rebase main`.
- [ ] Avoid mixing whitespace changes with code changes in the same commit. These make diffs harder to read and conflicts more likely.
- [ ] You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- [ ] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/Crm14Mapper.java [hunk 1]
```diff
@@ -716,4 +717,15 @@ default String convertCaseType(Crm14DetailsModel model) {
         }
         return "Unknown";
     }
+    @Named("convertPartnerInvolved")
+    default String convertPartnerInvolved(String s) {
+        if  (s == null || s.isEmpty())
+            return null;
+        return switch (Integer.parseInt(s)) {
```
Comment (gpjustice) [inline equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/Crm14Mapper.java:724]: Can we put case 1, 2 and 3 values in a const string? `private final static string`

Comment (solomonmanik) [reply equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/Crm14Mapper.java:727]: done

File: equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/Crm14Mapper.java [hunk 2]
```diff
@@ -3,11 +3,15 @@
 import org.apache.commons.lang3.StringUtils;
 import org.mapstruct.Mapper;
 import org.mapstruct.Mapping;
+import org.mapstruct.Named;
 import uk.gov.justice.laa.crime.equinity.historicaldata.generated.dto.*;
 import uk.gov.justice.laa.crime.equinity.historicaldata.model.*;
 
 @Mapper(componentModel = "spring")
 public interface Crm14Mapper extends CrmMapper {
+    final static String partnerInvolved_opt1="Co-defendant";
```
Comment (gpjustice) [inline equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/Crm14Mapper.java:12]: `private static final String PARTNER_INVOLVED_OPT1 ="Co-defendant";`

string variable should be in Capital `PARTNER_INVOLVED_OPT1`

Comment (solomonmanik) [reply equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/Crm14Mapper.java]: done.

**Review by tanovellino — APPROVED** [review]

**Pull request merged**